src/develop/ui/patterns/mobile/carousel.md

```diff
@@ -5,46 +5,42 @@ summary:
 
 # Carousel Pattern
 
-The Carousel Pattern
-enables you to display multiple items in a horizontal slide. Elements inside
-the placeholder will be split into different items. For dynamic content, use a
-List directly inside the placeholder. The Carousel block is ideal to create
-horizontal slides in smaller screens.
-
-Here's the preview in Service Studio:
+You can use the Carousel UI Pattern to display multiple items in a horizontal slide. This pattern is ideal for creating horizontal slides in smaller screens. You can also use this pattern for dynamic content, by placing a List directly inside the placeholder. 
 
 ![](images/Carousel_studio_preview.png)
 
-## How to Use the Carousel Pattern
+## How to use the Carousel UI Pattern
+
+1. In Service Studio, in the Toolbox, search for `Carousel`.
+  
+     The Carousel widget is displayed.
+
+    ![](images/carousel-widget.png)
+  
+
+1. From the Toolbox, drag the Carousel widget onto your application's screen. 
```

Comment:
We never explain where Carousel Items is - we just drag the Carousel widget that creates the "Carousel Items" thing in silence. How about "This will create the Carousel widget in the screen, with a "Carousel Items" container for you to add the content that will show in the Carousel"?